src/Sulu/Bundle/MediaBundle/Resources/js/containers/MediaCollection/MediaCollection.js

```diff
@@ -97,6 +97,8 @@ class MediaCollection extends React.Component<Props> {
         const securable = !locked
             && (permissions.security !== undefined ? permissions.security : MediaCollection.securable);
 
+        const hasCollectionId = ['string', 'number'].includes(typeof collectionStore.id);
```

Comment:
would use the same check as in other cases used `!collectionStore.id` so I would remove this variable. see next comment.

Reply:
Also this will be fixed with https://github.com/sulu/sulu/issues/5038 where we got also the design today.